### PR TITLE
Bump oas3

### PR DESCRIPTION
**Pull request by supertong**
Watch out for warnings like this when bumping. 
image

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [16.x, 18.x]
```
Comment (lukiano) [inline .github/workflows/ci.yml:13]: Should we include 20?

File: src/index.ts
Comment (lukiano) [inline src/index.ts:1]:
```
// Store coerced values
      ctx.oas = {
        request: {
          query: validRequest.query,
          params: validRequest.path,
          header: validRequest.header,
        },
        operationId: validRequest.operationId
      };
```
It seems that we're missing describing `ctx.oas` in the readme file.

**Review by lukiano — COMMENTED** [review]
LGTM

**Comment by supertong** [comment]
Thanks @lukiano +++++ I will address those in a following up PR.

**Pull request merged**